### PR TITLE
(#728) update omnisharp on every update of cake.bakery

### LINKED ISSUE #728: `omnisharp.json` is not updated after manually calling the "Update Intellisense support" command
When `bakeryPath` in `omnisharp.json` was modified manually, calling `Update Intellisense support` in *cake-vscode* will download a new/updated version of Cake.Bakery, but it will not fix the `bakeryPath` to point to this new installation.

### PR DESCRIPTION
**Pull request by nils-a**
fixes #728

### CODE REVIEW

File: src/bakery/cakeBakeryCommand.ts [hunk 1]
```diff
@@ -1,14 +1,9 @@
...
 
 export async function updateCakeBakeryCommand(extensionPath: string) {
-    // Make sure that we're in the correct place.
-    if (workspace.rootPath === undefined) {
-        window.showErrorMessage('You have not yet opened a folder.');
-        return;
-    }
-
     // Install Cake Bakery
     var result = await installCakeDebug(extensionPath);
```
Comment (bjorkstromm) [inline src/bakery/cakeBakeryCommand.ts:8]: rename this method while we are modifying this file :)

**Review by bjorkstromm — DISMISSED** [review]
One minor... How do I test it? Do we publish VSIX on PR build?

File: src/bakery/cakeBakeryCommand.ts (hunk 1 above)
Comment (nils-a) [reply src/bakery/cakeBakeryCommand.ts:8]: @bjorkstromm something like `installCakeDebug` --> `installCakeBakery` ?

**Comment by nils-a** [comment]
checkout, `build.ps1`, in VSCode: "Extensions: Install from VSIX". 

We should probably look into uploading the VSIX as an artifact, or publishing previews to https://www.vsixgallery.com (though I'm not sure it hosts vscode VSIX)

**Review by gep13 — APPROVED** [review]
LGTM!

**Review by gep13 — APPROVED** [review]
LGTM!

**Comment by gep13** [comment]
@nils-a thank you very much for getting this fixed up!

**Pull request merged**